### PR TITLE
Add basic functionality

### PR DESCRIPTION
**Pull request by ri-nath**

### CODE REVIEW

File: src/components/UserDisplay.js [hunk 1]
```diff
@@ -7,22 +7,23 @@ export default class UserDisplay extends Component {
 
         this.state = {
             name: '',
-            id: ''
+            id: '',
```
Comment (ri-nath) [inline src/components/UserDisplay.js:16]: Not sure why i did this

File: src/components/UserDisplay.js [hunk 2]
```diff
@@ -7,22 +7,23 @@ export default class UserDisplay extends Component {
...
         };
 
         this.handleChange = this.handleChange.bind(this);
-        this.onSubmit = this.onSubmit.bind(this)
+        this.onSubmit = this.onSubmit.bind(this);
```
Comment (ri-nath) [inline src/components/UserDisplay.js:20]: or this

**Comment by kaijchang** [comment]
## To Do:

- ~~Persist storage~~
- ~~Reset timer on new session~~
- ~~Show some sort of success message on sign in/out along with user info such as hours~~
- Add robotics specific stuff like friday meetings and build season
- ~~Google sheets~~

**Comment by kaijchang** [comment]
Wow i dont know how to do that. Rishi u are So Smart.

**Review by kaijchang — APPROVED** [review]
Let's merge for now.

**Comment by ri-nath** [comment]
Zoo wee mama

**Pull request merged**